typing/typedecl.ml

```diff
@@ -202,12 +202,12 @@ let make_params env params =
 
 let has_global_attr attributes =
   List.exists
-    (fun attr -> String.equal attr.attr_name.txt "ocaml.global")
+    (fun attr -> String.equal attr.attr_name.txt "extension.global")
     attributes
 
 let has_nonlocal_attr attributes =
   List.exists
-    (fun attr -> String.equal attr.attr_name.txt "ocaml.nonlocal")
+    (fun attr -> String.equal attr.attr_name.txt "extension.nonlocal")
```

Comment:
Can we move these functions to `Builtin_attributes`? It would be good to only need to change one place

Reply:
Also this should support both `ocaml.global` and `extension.global`